lib/c/criu.c

```diff
@@ -1793,3 +1793,21 @@ void criu_set_pidfd_store_sk(int sk)
 {
 	criu_local_set_pidfd_store_sk(global_opts, sk);
 }
+
+int criu_local_set_network_lock(criu_opts *opts, enum criu_network_lock_method method)
+{
+	opts->rpc->has_network_lock = true;
+	if (method == CRIU_NETWORK_LOCK_IPTABLES) {
+		opts->rpc->network_lock = (CriuNetworkLockMethod)method;
+		return 0;
+	} else if (method == CRIU_NETWORK_LOCK_NFTABLES) {
+		opts->rpc->network_lock = (CriuNetworkLockMethod)method;
+		return 0;
+	}
+	return -1;
```

Comment:
This could be simplified to something like:
```C
int criu_local_set_network_lock(criu_opts *opts, enum criu_network_lock_method method)
{
	int ret = -1;

	if (method == CRIU_NETWORK_LOCK_IPTABLES || method == CRIU_NETWORK_LOCK_NFTABLES) {
		opts->rpc->has_network_lock = true;
		opts->rpc->network_lock = (CriuNetworkLockMethod)method;
		ret = 0;
	}

	return ret;
}
```